examples/stabilizer_code.py

```diff
@@ -0,0 +1,340 @@
+from typing import List, Tuple
+
+import numpy as np
+
+import cirq
+
+# Based on:
+# Stabilizer Codes and Quantum Error Correction
+# Daniel Gottesman
+# https://thesis.library.caltech.edu/2900/2/THESIS.pdf
+#
+# However, there were some mistakes in the thesis:
+# https://www2.perimeterinstitute.ca/personal/dgottesman/thesis-errata.html
+
+
+def _BuildByCode(mat: np.ndarray) -> List[str]:
+    """
+    Takes into input a matrix of Boolean interpreted as row-vectors, each having dimension 2 * n.
+    The matrix is converted into another matrix with as many rows, but this time the vectors
+    contain the letters I, X, Y, and Z representing Pauli operators.
+    """
```

Comment:
Most (all?) cirq code follows the style in which docstring consists of succinct one-line description of the function, followed by empty line, followed by elaborated description, followed by empty line and args and return sections (where the elaborated description and args/returns sections are optional). Please look around cirq to see what this looks like in practice and try to stay consistent. Here is a silly example:

```python
def blah(size: int) -> Blah:
    """Computes blah.

    Blah is meaningless.

    Args:
        size: Size of the blah to compute.

    Returns:
        Blah of a specified size.
    """
```

Reply:
Done.